styles.css

```diff
@@ -166,4 +166,51 @@ a {
 
 .signin {
   text-align: left;
-}
\ No newline at end of file
+}
+
+/* style for login session */
+.login-session {
+  display: grid;
+  place-items: center;
+  margin: 0;
+  padding: 0;
+}
+.login-session > h2 {
+  margin: 0 0 12px;
+  font-size: 36px;
+  font-weight: 600;
+}
+.login-session > h3 {
+  margin: 0 0 30px;
+  font-weight: 500;
+  font-size: 16px;
+  color: rgba(0, 0, 0, 38);
+}
+.login-form {
+  width: 100%;
+  max-width: 500px;
+  margin: 0;
+  display: grid;
+  gap: 16px;
+}
+.login-form > a {
+  color: blue;
+}
+.login- form > button {
+  width: 100%;
+  height: 56px;
+  padding: 0 16px;
+  border-radius: 1px;
+  border: 0;
+  font-weight: 600;
+  color: #f9f9f9;
+}
+.login form > input {
+  border: 1px solid #353333;
+  margin-top: 20px;
+}

```

Comment:
Use the components.css file styles like we used on the Register page in order to achieve consistency.